container/openejb-core/pom.xml

```diff
@@ -712,6 +712,25 @@
       <scope>provided</scope>
     </dependency>
 
+    <dependency>
+      <groupId>org.eclipse.microprofile.config</groupId>
+      <artifactId>microprofile-config-api</artifactId>
+      <version>1.2</version>
+      <scope>compile</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.eclipse.microprofile.jwt</groupId>
+      <artifactId>microprofile-jwt-auth-api</artifactId>
+      <version>1.1-SNAPSHOT</version>
```

Comment:
1.1-SNAPSHOT?

Is this for the fixed TCK? I'm not aware of any plans for an actual release of 1.1 so this should probably be 1.0.